amplipi/app.py

```diff
@@ -163,6 +164,22 @@ class params(SimpleNamespace):
 api = SimplifyingRouter(dependencies=[Depends(CookieOrParamAPIKey)])
 
 
+@api.get('/collected_stats')
+@api.get('/collected_stats/')
+def get_collected_stats():
+  """Get the current contents of statcollector output file after filtering out any empty stream data"""
+  survey = dict(statcollector.UsageSurveySchema.load_from_disk())
+  return {header: data for header, data in survey.items() if data != statcollector.StreamUsageSchema()}
+
+
+@api.post('/collected_stats')
+@api.post('/collected_stats/')
+def post_collected_stats():
+  """Send the contents of the statcollector output file to AmpliPi devs"""
+  survey = statcollector.UsageSurveySchema.load_from_disk()
+  survey.phone_home()
+
+
```

Comment:
These do not belong at the top of this file
I do not know where they belong, but it isn't the top of the top
They stay here in the meantime because it's easier to not need to scroll too much to edit imports if need be during devwork

Reply:
These stats are effectively metadata about the api. Putting them in the API feels weird. That said keeping them outside of the api may make accessing them a little more awkward.